[Android] WebView: The element type "meta" must be terminated by the matching end-tag "</meta>"

<!--- Provide a brief summary of the issue -->
Running HelloFX on Android with a WebView and loading the following valid HTML content:
```
<html><head><meta charset="UTF-8"><title>Title</title></head><body>test</body></html>
```

produces an exception that prevents passing the `org.w3c.dom.Document` object to the WebEngine::documentProperty
```
[Tue Mar 07 13:23:43 CET 2023][INFO] [SUB] D/GraalCompiled(20108): [Fatal Error] :1:63: The element type "meta" must be terminated by the matching end-tag "</meta>".
[Tue Mar 07 13:23:43 CET 2023][INFO] [SUB] D/GraalCompiled(20108): Error parsing html: The element type "meta" must be terminated by the matching end-tag "</meta>".
[Tue Mar 07 13:23:43 CET 2023][INFO] [SUB] V/GraalActivity(20108): LOAD onReceivedHttpError: request: GET
[Tue Mar 07 13:23:43 CET 2023][INFO] [SUB] V/GraalActivity(20108): LOAD onReceivedHttpError: errorResponse: 
[Tue Mar 07 13:23:43 CET 2023][INFO] [SUB] E/GraalGluon(20108): nativeFailedURL called. Invoke method on webView
```

JavaFX WebEngine [parses](https://github.com/openjdk/jfx/blob/master/modules/javafx.web/src/ios/java/javafx/scene/web/WebEngine.java#L898) HTML (HTML-5 with valid void tags) into XHTML (without valid void tags) with:
```
DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
DocumentBuilder builder = factory.newDocumentBuilder();
Document document = builder.parse(new InputSource(new StringReader(pageContent)));
```        
and fails with such exception:

```
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 57; The element type "meta" must be terminated by the matching end-tag "</meta>".
        at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:262)
        at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:342)
        at com.gluonhq.web.HelloWeb.start(HelloWeb.java:29)
```

### Expected Behavior
<!--- Tell us what should happen -->

### Current Behavior
<!--- Tell us what happens instead of the expected behavior -->

### Steps to Reproduce
<!--- Steps to reproduce this bug -->

### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

--- a/src/main/resources/native/android/android_project/app/src/main/java/com/gluonhq/helloandroid/NativeWebView.java
+++ b/src/main/resources/native/android/android_project/app/src/main/java/com/gluonhq/helloandroid/NativeWebView.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2022, Gluon
+ * Copyright (c) 2020, 2023, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by

--- a/src/main/resources/native/android/android_project/app/src/main/java/com/gluonhq/helloandroid/NativeWebView.java
+++ b/src/main/resources/native/android/android_project/app/src/main/java/com/gluonhq/helloandroid/NativeWebView.java
@@ -81,7 +81,8 @@ public class NativeWebView {
                     @Override
                     public void onPageFinished(WebView view, final String url) {
                         Log.v(TAG, "Page finished: " + url);
-                        NativeWebView.this.webView.evaluateJavascript("document.documentElement.innerHTML", new ValueCallback<String>() {
+                        // convert HTML into XML before it can be parsed into DOM
+                        NativeWebView.this.webView.evaluateJavascript("new XMLSerializer().serializeToString(document)", new ValueCallback<String>() {
                             @Override
                             public void onReceiveValue(String s) {
                                 Properties p = new Properties();

